Testreglar/1.3.1/Nett/nett-1.3.1b.json

```diff
@@ -0,0 +1,467 @@
+{
+	"namn": "Nett-1.3.1b Visuelle tabeller er korrekt kodet 2023",
+	"id": "nett-1.3.1b",
+	"testlabId": 484,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Innhold og relasjoner i visuelle tabeller er korrekt kodet med:</p>\r\n<ul>\r\n<li>Tabelltittel</li>\r\n<li>Overskriftsceller\r\n<ul>\r\n<li>Tabeller med overskrift i kun en rad eller kolonne</li>\r\n<li>Tabeller med overskriftsceller i både kolonne og rad</li>\r\n<li>Tabeller med overskriftsceller som er felles for flere kolonner og/eller rader inne i samme tabell</li>\r\n</ul>\r\n</li>\r\n<li>Tabell brukt til layout skal kodes slik at den ignoreres av hjelpemiddelteknologi</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har nettsiden visuelle tabeller?",
+			"ht": "<p>Gjør en visuell inspeksjon og sjekk om det finnes visuelle tabeller på testsiden.</p>\n<p>Informasjon er tabuler når det er logiske relasjoner mellom tekst, tall eller annen data i to dimensjoner (vannrett og loddrett).</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden har ikke visuelle tabeller."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Hvilken tabell tester du?",
+			"ht": "<p>Beskriv tabellen, slik at det er mulig å identifisere den i ettertid. Hvis det gjelder flere tabeller, registrerer du en og en.</p>",
+			"type": "tekst",
+			"label": "Tabell:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Er tabellen brukt til layout av innhold?",
+			"ht": "<p>En tabell er brukt til layout når tabell-formatet er brukt for å oppnå et bestemt utseende/design på innholdet, og ikke for å presentere logiske relasjoner mellom data.</p>\n<ul>\n<li>Eksempel: tekst og bilder er kodet som tabell for å oppnå en bestemt avstand mellom tekst og bilder.</li>\n</ul>\n<p><strong>Merk:</strong> Det kan gjelde for hele nettsiden eller deler av innholdet på nettsiden.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"F46"
+			],
+			"ruting": {
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				},
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				}
+			}
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Er layout-tabellen kodet slik at den ignoreres av hjelpemiddelteknologi?",
```

Comment:
Kommer ikke innholdet i layout tabellen til å arve rollen og bli skjult det og? Er dette meningen?

Reply:
Vi lagte et par steg om dette, men etter diskusjon med Geir Sindre ble vi enig om at nettleser og hjelpemiddelteknologi tolker slik arving forskjellig. Derfor hvis aria role presentation ble er brukt feil, så er det brudd uansett. Bør vi ha slik steg hva synes du? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role

Reply:
Vi vurderer om det er behov for et merknad om dette eller steg for testing.

Reply:
Vi beholder denne som den er, dette er ikke en standard situasjon.